<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewProject</class>
 <widget class="QDialog" name="CreateNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">    QDialog {
        border-radius: 20px;
        background-color: rgba(61, 71, 112, 204);
        border: 1px solid rgba(255, 255, 255, 51);
    }
QWidget {
    font-family: &quot;思源黑体&quot;;
	color: white;
	background: transparent;
}
QLineEdit{
opacity: 0.5;
border-radius: 10px;
background: rgba(82, 93, 128, 1);
padding-left: 18px;   /* 清除默认padding */
padding-right: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralwidget" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#centralwidget{
opacity: 0.5;
border-radius: 20px;
background: rgba(61, 71, 112, 0.8);
}

QPushButton {
border:none;
background:transparent;
}

QToolButton {
border:none;
background:transparent;
}

QLabel {
border:none;
background:transparent;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_title" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#widget_title{
	background: qlineargradient(
	  x1:0, y1:0.5, x2:1, y2:0.5,  /* 90度水平渐变 */
	  stop:0 rgba(78, 228, 224, 0),      /* 0%位置 */
	  stop:0.1919 rgba(77, 153, 245, 64), /* 19.19%位置(0.25×255≈64) */
	  stop:1 rgba(78, 218, 227, 0)       /* 100%位置 */
	);
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../assets/dialog/new_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>思源黑体</family>
             <pointsize>21</pointsize>
            </font>
           </property>
           <property name="text">
            <string>新建项目</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>55</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>73</number>
        </property>
        <property name="bottomMargin">
         <number>48</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>22</number>
        </property>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string> 备注:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_project_route">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>请输入采集路线</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string> 创建人:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>项目名称:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string> 采集路线:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_project_name">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>请输入项目名称</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QTextEdit" name="textEdit_remark">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 10px;
background: rgba(82, 93, 128, 1);</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_creator">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>请输入创建人</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgba(255, 87, 51, 1);</string>
          </property>
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>100</number>
        </property>
        <property name="leftMargin">
         <number>138</number>
        </property>
        <property name="rightMargin">
         <number>138</number>
        </property>
        <property name="bottomMargin">
         <number>46</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_Accepted">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(73, 126, 229, 1),
        stop:1 rgba(78, 218, 222, 1));</string>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Cancel">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>思源黑体</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">    border-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 rgba(250, 77, 100, 1),
        stop:1 rgba(240, 112, 79, 1));</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateNewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>505</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
